### ISSUE TITLE
The "X" and "Reload" icons on the app's internal browser aren't visible if Bold Text is enabled

### ISSUE THREAD

**Issue by nellofn** [wordpress-mobile/WordPress-iOS#9396]
A user reported in 280611-h that when accessing a page within the app's internal browser, both the "X" and "Reload" icons on the top left and right corner, respectively, are there, working, but they are not visible.

### Expected behavior

I expected the buttons to be darker or bolder, but visible.

![screen shot 2018-05-17 at 5 02 23 pm](https://user-images.githubusercontent.com/17755234/40203727-1c2cfade-59f4-11e8-8cd0-eed07c21b4b7.png)

### Steps to reproduce: 

On the iPhone: 

- Go to Settings > General > Accessibility > Bold Text and turn it on. 
![screen shot 2018-05-17 at 5 00 15 pm](https://user-images.githubusercontent.com/17755234/40203623-cadd53c2-59f3-11e8-8139-c86c42dd9d76.png)

- It will prompt to restart the phone, go ahead and tap on "Continue" to restart the phone.
- Reopen the WordPress app, go to the Reader, click on any result and then click on a link to open the page in the app's internal browser. 
- Alternatively, go to My Sites > "Pick any site" > External > and tap on "View Site" to open the site in the app's internal browser.

Result: 

![screen shot 2018-05-17 at 4 59 23 pm](https://user-images.githubusercontent.com/17755234/40203598-b7b6d50c-59f3-11e8-818c-f7d9c699f016.png)

**Comment by elibud** [comment]
@etoledom can you look into this one please?

**Comment by etoledom** [comment]
Sure thing!

**Comment by etoledom** [comment]
This seems to be an iOS bug

> Apple Technical Support has confirmed that this is a bug in iOS 11. I've filed ID 34950920 in Bug Reporter.
([from here](https://forums.developer.apple.com/thread/89337))

I'll try to find a temporal workaround, but it might be a bit too hacky.
